ansible/roles/xl_auth/templates/xl_auth.service.j2

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=xl_auth daemon
+After=syslog.target network.target
+
+[Service]
+User=xl_auth
+Group=xl_auth
+WorkingDirectory=/opt/xl_auth
+Environment="PATH=/sbin:/bin:/usr/sbin:/usr/bin:/opt/xl_auth/venv/bin"
+Environment="FLASK_APP=autoapp.py"
+Environment="SERVER_NAME={{ xl_auth_server_name }}"
+Environment="XL_AUTH_SECRET={{ xl_auth_secret | default('not_secret, as fallback for `vagrant up`') }}"
+Environment="PREFERRED_URL_SCHEME=https"
+Environment="FLASK_DEBUG=0"
+Environment="SQLALCHEMY_DATABASE_URI=postgresql://{{ postgresql_user }}:{{ postgresql_password }}@127.0.0.1/{{ postgresql_db }}"
```

Comment:
@mxtthias Gets templated out incorrectly with "postgres" instead of "prod" as DB name. Not clear why. Fix in #192? Or #178.